### DISCUSSION TITLE
Indoor/Interior map

### DISCUSSION THREAD

**Discussion by YohanSciubukgian** [protomaps/basemaps#226] [Q&A]
I would like to create a map based on pmtile that could include:
- The whole planet map
- indoor/interior map (such as train station, airports, etc..)

I found out that you are providing daily/weekly planet map [Builds | Protomaps Basemaps](https://maps.protomaps.com/builds/) based on the [onthegomap/planetiler: Flexible tool to build planet-scale vector tilesets from OpenStreetMap data fast (github.com)](https://github.com/onthegomap/planetiler) project.

However, I didn't find any information if the pmtile protocol could support indoor/interior maps.
Do you support indoor/interior maps as part of the pmtiles protocol?
If yes, do you include it as part of the automated [Builds | Protomaps Basemaps](https://maps.protomaps.com/builds/)?
if yes, do you have a sample on how to access different level of an interior map (-1, 0, 1, ..) for an airport or a train station using a pmtiles ?

**Comment by bdon** [comment]
If there is indoor data available in OpenStreetMap, than it may be included. The pmtiles format simply stores whatever data you choose to put into it, it is not opinionated as to whether it is indoor/outdoor data etc.
